.github/workflows/zombie-bite.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  workflow_dispatch:
+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+  ci:
+    runs-on: parity-default
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - uses: extractions/setup-just@v2
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: 1.84.1
+          cache: false
+          components: rust-src
+      - name: install_zombie-bite
+        run: |
+          cargo install --git https://github.com/pepoviola/zombie-bite --bin zombie-bite
+      - name: zombie_bite
+        env:
+          ZOMBIE_BITE_CI_PATH: "/tmp/ci"
+          RUSTFLAGS: "-A warnings"
+        run: |
+          just run-zombie-bite
```

Comment:
Does this automatically exit once the code has been overridden and the snapshot created? from what I can see this would just continue running until the timeout is hit

Reply:
Oh sorry, sat on this for too long - didn't realise it was already merged